designer/client/cypress/e2e/process.cy.ts

```diff
@@ -153,6 +153,18 @@ describe("Process", () => {
 
         it("should have counts button and modal", () => {
             cy.viewport("macbook-15");
+            cy.contains(/^deploy$/i).click();
```

Comment:
Can you explain to me all of this code? Why can't we just place `cy.get("[data-testid=window]").matchImage();` right before `cy.deployScenario();`?

Reply:
inside deployScenario(); we open and close that window

Reply:
OK, so maybe we can paste an argument like, `withScreenshot`, and move this `cy.get("[data-testid=window]").matchImage();` inside of the `cy.deployScenario();` ?

Reply:
fix